<ui version="4.0" >
 <class>TriView</class>
 <widget class="QWidget" name="TriView" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="horizontalSpacing" >
    <number>1</number>
   </property>
   <property name="verticalSpacing" >
    <number>1</number>
   </property>
   <item row="0" column="1" >
    <widget class="QSlider" name="xSlicer" >
     <property name="maximum" >
      <number>0</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2" >
    <widget class="QSlider" name="zSlicer2" >
     <property name="maximum" >
      <number>0</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QSlider" name="ySlicer" >
     <property name="maximum" >
      <number>0</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="ImageStack" name="topView" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="ImageStack" name="sideView" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item rowspan="2" row="2" column="0" >
    <widget class="QSlider" name="zSlicer" >
     <property name="maximum" >
      <number>0</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="ImageStack" name="frontView" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageStack</class>
   <extends>QLabel</extends>
   <header location="global" >imagestack.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>zSlicer</sender>
   <signal>valueChanged(int)</signal>
   <receiver>zSlicer2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>14</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel" >
     <x>322</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zSlicer2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>zSlicer</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>434</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
